README.md

```diff
@@ -1,149 +1,194 @@
+Try it now: [https://sgrep.live](https://sgrep.live/)
 
-## Quick Examples
+## Overview
 
-| **pattern** | **will match code like** |
+Simple patterns for finding code in many languages:
+
+| **Pattern** | **Matches** |
 | :--- | :--- |
 | `$X == $X` | `if (node.id == node.id): ...` |
-| `foo(kwd1=1, kwd2=2, ...)` | `foo(kwd2=2, kwd1=1, kwd3=3)` |
-| `subprocess.Popen(...)` | `import subprocess as s; s.Popen(['foo'])` |
-
-→ [see more examples in the sgrep-rules registry](https://github.com/returntocorp/sgrep-rules)
+| `requests.get(..., verify=False, ...)` | `requests.get(url, timeout=3, verify=False)` |
+| `os.system(...)` | `from os import system; system('echo sgrep')` |
 
-## Supported Languages
+Language support:
 
-| **javascript** | **python** | **go** | **java** | **c** | **ruby** | **scala** |
+| **Python** | **Javascript** | **Go &nbsp; &nbsp; &nbsp;** | **Java &nbsp;** | **C &nbsp; &nbsp; &nbsp; &nbsp;** | **Typescript** | **PHP &nbsp; &nbsp;** |
 | :--- | :--- | :--- | :--- | :--- | :--- | :--- |
-| ✅ | ✅ | ✅ | ✅ | ✅ | coming | coming |
-
-→ see full language support details in [matrix.md](docs/matrix.md)
-
-## Meetups
+| ✅ | ✅ | ✅ | ✅ | ✅ | Coming... | Coming... |
 
-Want to learn more about sgrep? Check out these [slides from the r2c February meetup](https://r2c.dev/sgrep-public2.pdf)
+## Installing
 
-## Installation
+For the best experience, simply install `sgrep` with [Docker](https://docs.docker.com/install/):
 
-Too lazy to install? Try out [sgrep.live](https://sgrep.live)
-
-### Docker
-
-`sgrep` is packaged within a [docker container](https://hub.docker.com/r/returntocorp/sgrep), making installation as easy as [installing docker](https://docs.docker.com/install/).
+```
+$ docker pull returntocorp/sgrep
+```
 
-### Mac \(alpha\)
+`sgrep` binaries can also be installed on OSX and Ubuntu.
 
-The brew install is WIP and will change significantly.
+### OSX
 
-```bash
-brew tap returntocorp/sgrep https://github.com/returntocorp/sgrep.git
-brew install sgrep-r2c
-# Due to a naming conflict, when installed via brew, it is named `sgrok`
-sgrok --help
+```
+$ brew tap returntocorp/sgrep https://github.com/returntocorp/sgrep.git
+$ brew install sgrep-r2c
+$ sgrep-lint --help
 ```
 
-## Quickstart
-
-```bash
-docker pull returntocorp/sgrep
-
-cd /path/to/repo
-# generate a template config file
-docker run --rm -v "${PWD}:/home/repo" returntocorp/sgrep --generate-config
+### Ubuntu
 
-# look for findings
-docker run --rm -v "${PWD}:/home/repo" returntocorp/sgrep
+```
+$ wget https://github.com/returntocorp/sgrep/releases/download/v0.4.9/sgrep-0.4.9-ubuntu-16.04.tgz
+$ wget https://github.com/returntocorp/sgrep/releases/download/v0.4.9/sgrep-0.4.9-ubuntu-16.04.tgz.sha256
+$ sha256sum --check sgrep-0.4.9-ubuntu-16.04.tgz.sha256
+$ tar xf sgrep-0.4.9-ubuntu-16.04.tgz
+$ ./sgrep-lint-files/sgrep-lint --help
 ```
 
-## Usage
-
-### Rule Development
+## Using
```

Comment:
From here down I tried to move away from the theory behind `sgrep` and comprehensive docs toward getting them up and running as soon as possible. Where appropriate we can link out to more comprehensive docs or resources.